src/rules/maxFileLineCountRule.ts

```diff
@@ -55,17 +54,13 @@ export class Rule extends Lint.Rules.AbstractRule {
     }
 
     public apply(sourceFile: ts.SourceFile): Lint.RuleFailure[] {
-        const ruleFailures: Lint.RuleFailure[] = [];
-        const ruleArguments = this.getOptions().ruleArguments;
-        const lineLimit: number = ruleArguments[0];
+        const lineLimit: number = this.ruleArguments[0];
         const lineCount: number = sourceFile.getLineStarts().length;
-        const disabledIntervals = this.getOptions().disabledIntervals;
-
-        if (lineCount > lineLimit && disabledIntervals.length === 0) {
```

Comment:
Moved error to the bottom, so a `tslint:disable` anywhere should work. Presumably no one uses `tslint:disable-line` for this rule.